spec/core/1.0.md

```diff
@@ -37,78 +38,142 @@ Analogously, when a property is said to **accept unsigned integer values**, this
 In both these cases, the **numerical value** of each such byte string is the decimal value of the sequence of digits in the byte string's value, possibly negated by a sign.
 The module specification defining the property may impose additional restrictions on the numerical value of the property.
 
+```abnf
+; cannot use string literals here (boolean = "f" / "t")
+; because those are case-insensitive in ABNF
+boolean = %x66 / %x74
+```
+
+When a property is said to **accept boolean values**, this means that the set of acceptable values for this property are the two single-byte strings `f` and `t`.
+The **logical value** of the property SHALL be "false" for the byte string `f` and "true" for the byte string `t`.
+
 Module specifications which use some or all of the property types defined in this section SHALL reference this section.
 The recommended way to do so is by including the following sentence near the start of the specification:
 
 > This document uses the predefined property types from `vt6/core1.0`.
 
 ## 3. Message types for `vt6/core1.0`
 
-### 3.1. The `core1.lifetime-end` message
+### 3.1. The `core1.client-make` message
 
-- Role: request (not answered)
-- Directionality: both (see below)
+```vt6
+(core1.client-make <client-id> <stdin-screen-id> <stdout-screen-id> <stderr-screen-id>)
+```
+
+- Role: request (answered by `core1.client-new`)
+- Number of arguments: four
+
+A client sends a `core1.client-make` message to inform the terminal about a client ID that it has chosen.
```

Comment:
In this instance the word 'client' isn't accurate enough. Lifetime would actually be better.